test-case/check-reboot.sh

```diff
@@ -150,7 +150,8 @@ $old_content
 
 $boot_once_flag
 export LOG_ROOT='$(realpath $LOG_ROOT)'
-export DMESG_LOG_START_LINE=$(wc -l /var/log/kern.log|awk '{print $1;}')
+setup_kernel_check_point
+export KERNEL_CHECKPOINT
```

Comment:
Can you add a comment explaining why this one needs the export when the others don't?

Wouldn't it be simpler to export KERNEL_CHECKPOINT always? If not why not?

Reply:
Sorry, this is a mistake from previously version, the `export KERNEL_CHECKPOINT` should be removed.